.evergreen/config.yml

```diff
@@ -1104,12 +1104,16 @@ axes:
   - id: libmongoc-version
     display_name: libmongoc version
     values:
-      - id: "1.17-latest"
-        display_name: "1.17 latest"
+      - id: "lowest-supported"
+        display_name: "Lowest (1.18.0)"
         variables:
-          LIBMONGOC_VERSION: "r1.17"
-      - id: "master"
-        display_name: "Upcoming release (master)"
```

Comment:
Oh, should this have been updated a while back? The master branch of PHPC hasn't supported r1.17 in some time.

Reply:
Yes, that was missed. I'll revisit our build queue after the release